### ISSUE TITLE
Additional Sort Features for Metadata Browser Configs

### ISSUE THREAD

**Issue by jamlung-ri** [OpenConceptLab/ocl_issues#688]
A couple of additional needs for the Sort feature have come up:
1. Multi-sort: Ability to sort by more than one field i.e. id -> name
1. Sort by Extras Attributes: Allow extras attributes to be sorted by

**Comment by jamlung-ri** [comment]
@snyaggarwal The priority for this has been bumped up since it'll help with a deliverable. Is it feasible to do this either this sprint or next sprint? I will put it in this sprint for now, but please feel free to bump it back if your plate is too full right now.

**Comment by jamlung-ri** [comment]
Moving to next sprint. Note that this is higher priority since it's part of an end-of-June deliverable.

**Comment by snyaggarwal** [comment]
@jamlung-ri Multiple attribute sort is supported now on recognized attributes.
 - You can add `sort: '-last_update,name',` in tab configuration in place of `sortAsc` or `sortDesc` to use it. 
 - `-` denotes desc
 For example:
 - sort by id asc and name desc -- `sort='id,-name'`
 - sort by updated on desc and name asc -- `sort=-'last_update,name'`
 
 @jamlung-ri I am looking at sorting with extras attributes. Will get back to you on its limitations soon.

**Comment by snyaggarwal** [comment]
@jamlung-ri Due to the dynamic nature of `extras`, right now the way we index `extras` prevents us from sorting on it.
We will have to do a lot more work to provide structure to the index of `extras` dynamically. This is the same reason why the search on `extras` is supported with limitations.

**Comment by jamlung-ri** [comment]
We discussed this on the OCL Dev call yesterday, and it was discussed that the approach to doing this in the near term would not be the final or best solution. We will wait on Jon's input before we implement this change, since we would only need it for the [Indicator Mapping](https://docs.google.com/document/d/10GydfRC7OM0tM7l6o2klSuiKQow4_3HRs6ZNyANbjbc/edit#) view.

@paynejd If you think that sorting the rows is absolutely necessary for the Indicator Mapping view right now, we can implement a fix that will take a couple of days. But unless you deem that necessary, we will just leave it out of the Indicator Mapping view for now until sorting by Extra attributes is supported.

**Comment by jamlung-ri** [comment]
A follow-up ticket was created to address the sorting by Extra attributes. Otherwise, this ticket is complete.
